easybuild/easyblocks/p/psmpi.py

```diff
@@ -92,6 +187,10 @@ def configure_step(self):
         if self.cfg['mpich_opts'] is not None:
             self.cfg.update('configopts', ' --with-mpichconf="%s"' % self.cfg['mpich_opts'])
 
+        # Add PGO related options, if enabled
+        if self.cfg['pgo']:
+            self.cfg.update('configopts', ' --with-profile=gen --with-profdir=%(builddir)s/profile')
```

Comment:
@damianam Avoid having the profdir location in two places, use a class variable that you define early on (in `__init__`)